.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -14,3 +14,6 @@ for each release artifact. Do not try to sign all of them at once by globbing. I
 
  - [ ] Create a draft release on GitHub and upload all the release artifacts and their signatures. Copy and paste the release notes from NEWS here as well.
  - [ ] Publish the release
+ - Update the `release` tag on Quay:
+   - [ ] Visit the [Quay tags page](https://quay.io/repository/coreos/fcct?tab=tags) and wait for a versioned tag to appear
+   - [ ] Click the gear next to the tag, select "Add New Tag", enter `release`, and confirm
```

Comment:
Does this procedure work even if `release` already exists?

Reply:
Yup!